packages/components/src/utils/colors-values.js

```diff
@@ -31,6 +31,7 @@ export const G2 = {
 		700: '#757575', // Meets 4.6:1 text contrast against white.
 		600: '#949494', // Meets 3:1 UI or large text contrast against white.
 		400: '#ccc',
+		300: '#ddd',
 		200: '#ddd', // Used for most borders.
```

Comment:
Hold up, now 200 and 300 are the same 🤔 

I guess the error is that 200 should be `#e0e0e0`:

<img width="562" alt="Screenshot 2022-01-13 at 09 24 29" src="https://user-images.githubusercontent.com/1204802/149293065-ed90b182-5116-4f23-b10b-ab35477c0901.png">

Since it looks like 200 was already erroneous before this PR, it seems fine to not fix that in this one, but this is an error we should correct.

Reply:
Yeah I noted that on the updated OP and agree, we should fix it separately.

Reply:
Opened https://github.com/WordPress/gutenberg/pull/38109 to address this separately

Reply:
#38109 has been merged, this PR will likely require a rebase to solve conflicts